static/app/components/modals/demoEndModal.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {fetchGuides} from 'sentry/actionCreators/guides';
+import {ModalRenderProps} from 'sentry/actionCreators/modal';
+import {navigateTo} from 'sentry/actionCreators/navigation';
+import {Client} from 'sentry/api';
+import Button from 'sentry/components/button';
+import ModalTask from 'sentry/components/onboardingWizard/modalTask';
+import {SidebarPanelKey} from 'sentry/components/sidebar/types';
+import {IconClose} from 'sentry/icons/iconClose';
+import {t, tct} from 'sentry/locale';
+import SidebarPanelStore from 'sentry/stores/sidebarPanelStore';
+import {useRouteContext} from 'sentry/utils/useRouteContext';
+
+type Props = ModalRenderProps & {orgSlug: string | null; tour: string};
+
+function DemoEndingModal({tour, closeModal, orgSlug}: Props) {
+  const api = new Client();
+
+  const routeContext = useRouteContext();
+  const {router} = routeContext;
```

Comment:
```suggestion
  const {router} = useRouteContext();
```

Is this right @NisanthanNanthakumar ?

Reply:
I don't want to answer for @NisanthanNanthakumar but according to a similar code we have in the app, this is correct https://github.com/getsentry/sentry/blob/a66d78d30e14f5e86645f1d6a91b296ffb7a09b6/static/app/components/onboardingWizard/task.tsx#L58